test/compilable/revert_dip1000.d

```diff
@@ -0,0 +1 @@
+// REQUIRED_ARGS: -revert=dip1000
```

Comment:
Maybe have some code to verify it doesn't trigger deprecation messages?

Reply:
Done.